onelogin_aws_cli/argparse.py

```diff
@@ -13,17 +14,41 @@ class OneLoginAWSArgumentParser(argparse.ArgumentParser):
     def __init__(self):
         super().__init__(description='Login to AWS with OneLogin')
 
+        env_config_name = os.getenv('ONELOGIN_AWS_CLI_CONFIG_NAME')
+        if env_config_name is not None:
+            default_config_name = env_config_name
+        else:
+            default_config_name = 'default'
+
         self.add_argument(
-            '-C', '--config-name', default='default', dest='config_name',
+            '-C', '--config-name',
+            default=default_config_name,
+            dest='config_name',
             help='Switch configuration name within config file'
         )
 
+        env_profile = os.getenv('ONELOGIN_AWS_CLI_PROFILE')
+        if env_profile is not None:
+            default_profile = env_profile
+        else:
+            default_profile = ''
+
         self.add_argument(
-            '--profile', default='', help='Specify profile name of credential'
+            '--profile',
+            default=default_profile,
+            help='Specify profile name of credential',
         )
 
+        env_username = os.getenv('ONELOGIN_AWS_CLI_USERNAME')
```

Comment:
Is it worth simplifying this down to `ONELOGIN_USERNAME`?

Reply:
It's generally a good idea to prefix the env vars with the unique program name. You never know what somebody's use case might be.